loops/3_loops_array.php

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env php
+
+<?php
+
+/*
+ * Create all kind of array loop. Each loop in it's own function with one parameter.
+ *
+ * Each loop should print() every single item in given array.
+ */
+
+$mainArray = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+/**
+ * @param (array) sets array to iterate
+ */
+function forEachLoop($array) {
+    foreach ($array as $arr) {
+        print("$arr\n");
+    }
+}
+
+/**
+ * @param (array) sets array to iterate
+ */
+function forLoop($array) {
+    $arrayLength = count($array);
+    for ($i = 0; $i < $arrayLength; $i++) {
```

Comment:
Brak konsekwencji.
Wszędzie używasz `$i += 1`, a tutaj nagle `$i++`.

Wybierz jedno i bądź konsekwentny.
Zalecam, żeby to jednak było `$i += 1`

W kolejnych pętlach popełniasz ten sam błąd.